tools/zbcat

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+# Contributed by Bryan Halfpap <Bryanhalf@gmail.com>, Copyright 2016
+import sys
+import argparse
+from time import sleep
+import logging
+from killerbee import *
+
+"""
+====================================
+ZBCAT - The 802.15.4 packet pusher!
+====================================
+Pushes 802.15.4 DATA packets onto the specified channel, to the specified PAN and Destination address from the defined source address.
+Can be configured to respond to DATA REQUEST packets (--respond) or it can just spew packets out to a target (maybe it wasn't requesting, but is still accepting)
+Written to help facilitate these situations due to a Wireless Village Wireless Capture-the-Flag at Shmoocon 2016 by @redbarondefcoin
+"""
+
+def create_packet(panid, source, dest, data, extsource=False, ack=False, shortaddress=True, seq=0):
+    if extsource == False:
+        extsource = ''
+
+    if shortaddress:
+        mode = 0b10
+    else:
+        mode = 0b11
+
+    # bit manipulating the FCS was the quickest way, but here I break down the bytes in the order they are pushed (starting with the left-most bit)
```

Comment:
since scapy-com isn't something everyone has installed and because doing bit-manipulation is fast (not that i've tested speed against the d15d4) I chose to just do it inside the lib.

I think that the best, most extendable manner of packet creation would be to handle it inside of killerbee, but that does have the drawback of not having a centralized packet processing library handle it.

My overall pref. would be to do it inside of killerbee to keep features and requirements moving swiftly, then port changes back into scapy-com.